clients/deltalake/core/src/main/scala/org/projectnessie/deltalake/NessieLogStore.scala

```diff
@@ -70,14 +71,34 @@ class NessieLogStore(sparkConf: SparkConf, hadoopConf: Configuration)
   var lastSnapshotUuid: Option[String] = None
 
   private val client: NessieClient = {
-    NessieClient.builder().fromConfig(c => hadoopConf.get(c)).build()
+    val catalogConfMap = catalogConf()
+    val removePrefix = (x: String) => x.replace("nessie.", "")
+    NessieClient
```

Comment:
This worries me.
`NessieClient` is `AutoCloseable`, but it's never closed. Maybe just refactor the `lazy val` usage.

Reply:
created a wrapper class. Not pretty but clearer